File: internal/content/testdata/bench/go/nc-house-robber.go
func rob(nums []int) int {
	if len(nums) == 0 {
		return 0
	}
	if len(nums) == 1 {
		return nums[0]
	}
	prev2, prev1 := nums[0], max(nums[0], nums[1])
	for i := 2; i < len(nums); i++ {
		curr := max(prev1, prev2+nums[i])
		prev2, prev1 = prev1, curr
	}
	return prev1
}

File: internal/content/testdata/bench/rust/nc-house-robber.rs
fn rob(nums: Vec<i64>) -> i64 {
    if nums.is_empty() {
        return 0;
    }
    if nums.len() == 1 {
        return nums[0];
    }
    let mut prev2 = nums[0];
    let mut prev1 = nums[0].max(nums[1]);
    for i in 2..nums.len() {
        let curr = prev1.max(prev2 + nums[i]);
        prev2 = prev1;
        prev1 = curr;
    }
    prev1
}
